lighthouse-core/lib/stack-packs.js

```diff
@@ -7,27 +7,39 @@
 
 const stackPacks = require('@lighthouse/stack-packs');
 
+const stackPacksToInclude = [{
+  packId: 'wordpress',
+  npmOrIconName: 'wordpress',
+  requiredStacks: ['js:wordpress']
+}];
+
 /**
  * @param {LH.Artifacts} artifacts
- * @return {Array<LH.StackPacks>}
+ * @return {Array<LH.StackPack>}
  */
 function getStackPacks(artifacts) {
-  /** @type {Array<LH.StackPacks>} */
+  /** @type {Array<LH.StackPack>} */
   const packs = [];
 
-  artifacts.Stacks.forEach(x => {
-    const stack = x;
-    stack.id = 'wordpress';
-    // artifacts.Stacks.forEach(stack => {
-    if (stackPacks[stack.id]) {
-      const pack = stackPacks[stack.id];
+  artifacts.Stacks.forEach(pageStack => {
+    /** @type {import('@lighthouse/stack-packs').Pack|undefined} */
+    let matchedPack;
+    stackPacksToInclude.some(stackPackToIncl => {
```

Comment:
seems like you can write this as a .find()

```js
// Is requiredStacks of the packs Lighthouse supports satisfied?
const safelistItem = stackPackSafelist.find(stackPackToIncl => 
  stackPackToIncl.requiredStacks.includes(`${pageStack.detector}:${pageStack.id}`))
);
if (!safelistItem) return;

// Grab the full pack definition
const matchedPack = stackPacks.find(pack => pack.id === safelistItem.npmOrIconName);
if (!matchedPack) throw new Error('A safelisted stackpack has no definition wtffff');

packs.push({ ...
```

(didn't try this code. could be wrong.)